src/locale/messages.da.xlf

```diff
@@ -488,14 +486,22 @@
         <source>keyboard-mapping-settings-page.delete-all-keyboard-mappings.success <x id="PH" equiv-text="actionTriggerIdsToBeDeleted.length"/></source>
         <target state="new"><x id="PH" equiv-text="actionTriggerIdsToBeDeleted.length"/> genveje blev slettet</target>
       </trans-unit>
-      <trans-unit id="6774239910313063167" datatype="html">
-        <source>keyboard-mapping.settings-page.invalid-field</source>
-        <target state="new">Ugyldig data, skal redigeres. </target>
-      </trans-unit>
       <trans-unit id="5805882241277564116" datatype="html">
         <source>keyboard-mapping-settings-page-import-keyboard-mappings.actions-missing</source>
         <target state="new">En eller flere importeret genveje havde en ugyldig action og blev derfor ikke importeret!</target>
       </trans-unit>
+      <trans-unit id="8483503986037571867" datatype="html">
+        <source>action-triggers.override-color.label</source>
+        <target state="new">Overskriv standard farve</target>
+      </trans-unit>
+      <trans-unit id="2313705565869821168" datatype="html">
+        <source>color-picker.choose-color.placeholder</source>
+        <target state="new">Vælg en farve</target>
+      </trans-unit>
+      <trans-unit id="4439615397476866844" datatype="html">
+        <source>action-triggers.override-color.help-text</source>
+        <target state="new">Vælg en ny farve hvis du gerne vil overskrive standard farven for genvejen.</target>
```

Comment:
The danish translation does not match the english in terms of `default color` vs `key color`.

Reply:
Updated to be `Overskriv genvejs farven` and `... hvis du gerne vil overskrive farven for genvejen`.